app/src/main/java/com/nevilleantony/prototype/adapters/CompletedListAdapter.java

```diff
@@ -0,0 +1,62 @@
+package com.nevilleantony.prototype.adapters;
+
+import android.content.Context;
+import android.content.Intent;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.nevilleantony.prototype.R;
+import com.nevilleantony.prototype.downloadmanager.DownloadService;
+
+import java.io.File;
+import java.util.List;
+
+public class CompletedListAdapter extends RecyclerView.Adapter<CompletedListAdapter.RecyclerViewHolder> {
+
+    private final List<File> downloads;
+    private final Intent downloadIntent;
+    private final Context context;
+
+    public CompletedListAdapter(Context context, List<File> completedList) {
+        this.downloads = completedList;
+        downloadIntent = new Intent(context, DownloadService.class);
```

Comment:
Do we need ```downloadIntent``` in completed page, this is for starting service in downloads page, right?